en/docs/guides/identity-lifecycles/extend-username-and-password-recovery.md

```diff
@@ -0,0 +1,534 @@
+
+The password recovery for of WSO2 Identity Server is as follows:
+
+1. User provides a set of unique claims to identify the user account.
+2. The API prompts the user to select a channel from the available notification channels for receiving the username recovery notification.
+3. The user selects a notification channel.
+4. The server sends the recovery notification to the user via the preferred notification channel.
+5. If the preferred channel is:
+    - **Email**: The user can click the reset link sent to the registered email address and reset the password.
+    - **SMS**: The user provides the received One-Time Password (OTP) and reset the password.
+
+The following sections walk you through configuring and trying out password recovery via SMS and Email channels.
+
+??? Warning "If you are migrating from IS 5.10.0 or above"
+    If you have migrated from IS 5.10.0 or above, ensure that the `IdentityMgtEventListener` with the `orderId=50` is set to `false` and that the Identity Listeners with `orderId=95` and `orderId=97` are set to `true` in the `<IS_HOME>/repository/conf/deployment.toml` file.
+
+    Note: You can skip this step if there are no entries for `event.default_listener.xxx` in the `deployment.toml` file.
+    
+    ``` java
+    [event.default_listener.identity_mgt]
+    priority= "50"
+    enable = false
+    [event.default_listener.governance_identity_mgt]
+    priority= "95"
+    enable = true
+    [event.default_listener.governance_identity_store]
+    priority= "97"
+    enable = true
+    ```
+
+## Set up password recovery
+
+Follow the steps given below to recover a user in the super tenant (i.e., `carbon.super`).
+
+1. Add the following properties to the `deployment.toml` file in the `IS_HOME/repository/conf` folder to configure the WSO2 Identity Server to send confirmation emails.
+
+    ``` toml
+    [output_adapter.email]
+    from_address= "wso2iamtest@gmail.com"
+    username= "wso2iamtest"
+    password= "Wso2@iam70"
+    hostname= "smtp.gmail.com"
+    port= 587
+    enable_start_tls= true
+    enable_authentication= true
+    ```
+
+    !!! Note
+        - Add this configuration only if you wish to configure WSO2 Identity Server to send confirmation emails. See [Configure the Email Sending Module]({{base_path}}/deploy/configure-email-sending/) for more information
+        - Alternatively, you can use your own email managing mechanism.
+
+
+2. Add the following properties to the `deployment.toml`.
+
+    ``` toml
+    [identity_mgt.notification_channel_recovery]
+    recovery_code_validity=2
+    resend_code_validity=5
+    
+    [identity_mgt.password_reset_sms]
+    sms_otp_validity=2
+    ```
+
+    <table>
+       <tr>
+           <th>Property Name</th>
+           <th>Functionality</th>
+       </tr>
+       <tr>
+          <td><code>recovery_code_validity</code></td>
+          <td>
+            <ul>
+                <li>Validity period of the recovery code given after initiating username/ password
```

Comment:
Should this be part of the list? Looks like this is a description followed by the unit and default value.

Reply:
I think this is correct. 

shown below is how its represented in the docs:

![image](https://user-images.githubusercontent.com/84217321/233777746-1ca4a9b0-bb12-41e5-a163-58e6a57413fc.png)

Reply:
But +1 to your suggestion as well. Will update it